core/src/components/button/button.ios.scss

```diff
@@ -34,7 +34,8 @@
  * can add the different font size/weight in a toolbar
  * but still let "size"/"strong" override it.
  */
-:host(.in-buttons) {
+:host(.in-buttons),
+:host(.in-buttons.button-clear) {
```

Comment:
This selector is necessary because the `.button-clear` class comes after this and overrides this class. Alternatively we could move this down below the button fills, while leaving it above the size/strong selectors.